packages/ssx-sdk/src/modules/Encryption.ts

```diff
@@ -51,14 +52,224 @@ interface IEncryption {
+   * {@link https://developer.litprotocol.com/resources/supportedChains Lit's docs} 
+   * for more information. 
+   * @param chainId - Unique chain id.
+   * @returns - The chain name follwing the Lit Protocol docs.
+   */
+  private getChainName = (chainId: number) => {
+    switch (chainId) {
+      case 1:
+        return "ethereum";
+      case 137:
+        return "polygon";
+      case 250:
+        return "fantom";
+      case 100:
+        return "xdai";
+      case 56:
+        return "bsc";
+      case 42161:
+        return "arbitrum";
+      case 43114:
+        return "avalanche";
+      case 1311:
+        return "fuji";
+      case 1666600000:
+      case 1666600001:
+      case 1666600002:
+      case 1666600003:
+        return "harmony";
+      case 42:
+        return "kovan";
+      case 80001:
+        return "mumbai";
+      case 5:
+        return "goerli";
+      case 3:
+        return "ropsten";
+      case 4:
+        return "rinkeby";
+      case 25:
+        return "cronos";
+      case 10:
+        return "optimism";
+      case 42220:
+        return "celo";
+      case 1313161554:
+        return "aurora";
+      case 955305:
+        return "eluvio";
+      case 44787:
+        return "alfajores";
+      case 50:
+        return "xdc";
+      case 9001:
+        return "evmos";
+      case 9000:
+        return "evmosTestnet";
+      case 97:
+        return "bscTestnet";
+      case 84531:
+        return "baseGoerli";
+      case 1284:
+        return "moonbeam";
+      case 1285:
+        return "moonriver";
+      case 1287:
+        return "moonbaseAlpha";
+      case 314:
+        return "filecoin";
+      case 3141:
+        return "hyperspace";
+      case 534353:
+        return "scrollAlphaTestnet";
+      case 324:
+        return "zksync";
+      case 245022934:
+        return "solana";
+      case 245022926:
+        return "solanaDevnet";
+      case 245022940:
+        return "solanaTestnet";
+      // case ?:  
+      //   return "cosmos";
+      // case ?:  
+      //   return "kyve";
+      // case ?:  
+      //   return "evmosCosmos";
+      // case ?:  
+      //   return "evmosCosmosTestnet";
+      // case ?:  
+      //   return "cheqd";
+      // case ?:  
+      //   return "juno"; 
+      default:
+        "";
+    }
+  }
+
+  public connect = async () => {
+    await this.client.connect();
+    this.litNodeClient = this.client;
+  }
+
+  public encrypt = async (data: LitEncryptData) => {
+    if (!this.litNodeClient) {
+      await this.connect();
+    }
+
+    const { session } = this.userAuth as UserAuthorization;
+
+    if (!session) {
+      throw new Error("Sign in before trying to encrypt your data.")
+    }
+
+    const authSig = {
+      address: session.address,
+      derivedVia: "web3.eth.personal.sign",
+      sig: session.signature,
+      signedMessage: session.siwe,
+    };
+
+    const { encryptedString, symmetricKey } = await LitJsSdk.encryptString(data.content)
```

Comment:
I'm only encrypting strings here, but they have [other methods](https://developer.litprotocol.com/SDK/Explanation/encryption#encrypting) to encrypt. We can add/change this call